### PR TITLE
feat(changelog): domains-and-dns-changed-increase-in-the-cost-of-the-fr- 2024-05-15

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: Maxime Besson

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

File: changelog/may2024/2024-05-15-domains-and-dns-changed-increase-in-the-cost-of-the-fr-.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    fullname: 'Join the #[xxx] channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-05-15
+category: network
+product: domains-and-dns
+---
+
+As you may see, the AFNIC announced that the .FR extension will increase its price by 11,1% on 1st March 2024 : https://www.afnic.fr/observatoire-ressources/actualites/le-fr-passe-de-4-56e-a-507e-a-partir-du-1er-mars-2024/
+
+This augmentation will also impact the cost of .FR domains you buy from Scaleway, starting June 1st 2024.
```
Comment (bene2k1) [inline changelog/may2024/2024-05-15-domains-and-dns-changed-increase-in-the-cost-of-the-fr-.mdx:14]:
```suggestion
Certainly! Here's a revised version:

As recently announced by AFNIC, the price of the .FR extension registration fee is about to increase by 11.1% effective from March 1st, 2024. You can find further details on this price increase [here](https://www.afnic.fr/observatoire-ressources/actualites/le-fr-passe-de-4-56e-a-507e-a-partir-du-1er-mars-2024/).

This price adjustment will subsequently influence the pricing of .FR domains purchased through Scaleway, commencing from June 1st, 2024.
```

File: changelog/may2024/2024-05-15-domains-and-dns-changed-increase-in-the-cost-of-the-fr-.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Increase in the cost of the .FR extension
+status: changed
+author:
+    fullname: 'Join the #[xxx] channel on Slack.'
```
Comment (RoRoJ) [inline changelog/may2024/2024-05-15-domains-and-dns-changed-increase-in-the-cost-of-the-fr-.mdx:5]:
```suggestion
    fullname: 'Join the #domains channel on Slack.'
```

**Review by RoRoJ — APPROVED** [review]

**Comment by bene2k1** [comment]
Do not merge this PR before GO from Maxime!

**Review by nerda-codes — APPROVED** [review]

File: changelog/may2024/2024-05-15-domains-and-dns-changed-increase-in-the-cost-of-the-fr-.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+status: changed
+author:
+    fullname: 'Join the #domains channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-05-15
+category: network
+product: domains-and-dns
+---
+
+As recently announced by AFNIC, the price of the .FR extension registration fee is about to increase by 11.1% effective from March 1st, 2024. You can find further details on this price adjustment [here](https://www.afnic.fr/observatoire-ressources/actualites/le-fr-passe-de-4-56e-a-507e-a-partir-du-1er-mars-2024/).
```
Comment (nerda-codes) [inline changelog/may2024/2024-05-15-domains-and-dns-changed-increase-in-the-cost-of-the-fr-.mdx:12]:
```suggestion
As recently announced by AFNIC, the price of the .FR extension renewal fee is about to increase by 11.1% effective from March 1st, 2024. You can find further details on this price adjustment [here](https://www.afnic.fr/observatoire-ressources/actualites/le-fr-passe-de-4-56e-a-507e-a-partir-du-1er-mars-2024/).
```

File: changelog/may2024/2024-05-15-domains-and-dns-changed-increase-in-the-cost-of-the-fr-.mdx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+title: Increase in the cost of the .FR extension
```
Comment (nerda-codes) [inline changelog/may2024/2024-05-15-domains-and-dns-changed-increase-in-the-cost-of-the-fr-.mdx:2]:
```suggestion
title: Increase in the cost of the .FR extension renewal
```

**Comment by nerda-codes** [comment]
Updated and merged upon Maxime's request.

**Pull request merged**